python/ray/air/tests/test_tracebacks.py

```diff
@@ -47,6 +47,27 @@ def test_short_traceback(levels):
     assert i == levels - start_traceback + 1
 
 
+def test_recursion():
```

Comment:
Any ideas on why Maximum recursion happens iff `pickling_support.install()` is called? Should we also test it here?

Reply:
Seems like a bug in `tblib`. 

Good point on testing. Originally I was going to remove `tblib` as a dependency in a separate PR, but even if I do I can add it as a test dependency.